.github/workflows/base-images.yaml

```diff
@@ -32,3 +32,9 @@ jobs:
           pushd images/git
             IMAGE=ghcr.io/${{ github.repository_owner }}/base-git docker buildx bake --push -f ../docker-bake.hcl
           popd
+
+      - name: Build Waiter
+        run: |
+          pushd images/waiter
+            IMAGE=quay.io/shipwright/base-waiter docker buildx bake --push -f ../docker-bake.hcl
```

Comment:
Please change this to ghcr.io.

Reply:
Thanks for the pointer, marking this as resolved.